src/components/ContactDetails.tsx

```diff
@@ -32,6 +35,7 @@ const ContactDetails = () => {
                 {zip} <br />
                 {country}
             </address> 
+            <Link to="/contacts/:id/update">Update Contact</Link>
```

Comment:
nit: it's more standard to be explicit with ID naming just to avoid confusion
```suggestion
            <Link to="/contacts/:contactID/update">Update Contact</Link>
```